docs/fundamentals/syslib-diagnostics/syslib0035.md

```diff
@@ -0,0 +1,14 @@
+---
+title: SYSLIB0035 warning - ComputeCounterSignature without specifying a CmsSigner is obsolete
+description: Learn about the obsoletion of the parameterless ComputeCounterSignature method that generates compile-time warning SYSLIB0035.
+ms.date: 09/07/2021
+---
+# SYSLIB0035: ComputeCounterSignature without specifying a CmsSigner is obsolete
+
+The <xref:System.Security.Cryptography.Pkcs.SignerInfo.ComputeCounterSignature?displayProperty=nameWithType> method is marked as obsolete, starting in .NET 6. Using this API in code generates warning `SYSLIB0035` at compile time.
+
+## Workaround
+
+Use the overload that accepts a <xref:System.Security.Cryptography.Pkcs.CmsSigner>, that is, <xref:System.Security.Cryptography.Pkcs.SignerInfo.ComputeCounterSignature(System.Security.Cryptography.Pkcs.CmsSigner)?displayProperty=nameWithType>.
```

Comment:
"that is", or "such as"?

Reply:
I think there's only one overload that applies, so "such as" wouldn't make sense to me.